test/integration_test.py

```diff
@@ -942,8 +945,13 @@ def list_source_ids():
                 self.assertIsSubset(indexed_source_ids, list_source_ids())
             with self._public_service_account_credentials:
                 public_source_ids = list_source_ids()
+            # An invalid nonempty oauth2 token produces the same
```

Comment:
```suggestion
            # An invalid nonempty OAuth 2.0 token produces the same
```

Reply:
This addition needs to be removed due to your [other comment](https://github.com/DataBiosphere/azul/pull/3879#discussion_r816406061).